backend_jobs/ingestion_pipeline/pipeline_lib/data_types.py

```diff
@@ -18,23 +18,31 @@
 import enum
 
 class ImageType(enum.Enum):
-    """
-    This enum represents the different imagery types.
+    """ This enum represents the different imagery types.
     """
     SATELLITE = 1
     DRONE = 2
     LIVE_FEED = 3
     CAMERA = 4
 
+class VisibilityType(enum.Enum):
+    """ This enum represents the different visibility types.
+    """
+    INVISIBLE = 0
+    VISIBLE = 1
+
 @dataclass
 class ImageAttributes():
     """ This class consists of the attributes we need to extract from each image.
     """
-    image_id: str
-    url: str
-    image_type: ImageType
-    date_shot: datetime
-    attribution: str
-    format: str
-    resolution: {} # type {'height':int, 'width':int}
-    coordinates: {} # type {'latitude':float, 'longitude':float}
+    image_id: str # Unique id of the image.
+    url: str # Url of the image, e.g.
+    #"https://live.staticflickr.com/3953/15317370767_d01a2327a9_c.jpg"
+    image_type: ImageType # Type of image, e.g camera, drone.
+    date_shot: datetime # Date the image was taken, e.g. 10/11/14, 5:12 PM
+    attribution: str # The attribution of the image, e.g. "Tal Tamir".
+    format: str # The format of the image, e.g. "jpg"
```

Comment:
I did not understand the question,
the format does not have restrictions- it can be JPEG or JPG,
I wrote JPG because that is what flickr provides.

e.g. https://live.staticflickr.com/{server-id}/{id}_{secret}_{size-suffix}.jpg